src/infrastructure/app/docs/ApiDocGenerator.ts

```diff
@@ -229,18 +232,13 @@ export class ApiDocGenerator implements IApiDocGenerator {
   }
 
   setServer(url: string, description: "Local server"): void {
-    if (this.env !== DEV || !Object.keys(this.apiDoc.paths).length) {
-      SchemasStore.dispose();
-      return;
-    }
 
     this.apiDoc.servers.push({
       url,
       description,
     });
 
     this.setSchemas(SchemasStore.get());
-    this.saveApiDoc();
-    SchemasStore.dispose();

```

Comment:
I think this is important to clean the unnecessary garbage, so pls include in your new changes.